frontend/src/component/personalDashboard/Grid.tsx

```diff
@@ -1,6 +1,75 @@
 import { Box, Grid, styled } from '@mui/material';
 import type { Theme } from '@mui/material/styles/createTheme';
 
+export const ContentGridContainer = styled('div')({
+    containerType: 'inline-size',
+});
```

Comment:
You need external containers to be able to use container queries. This contains the grid so we can check the size of it.

Reply:
why only inline and not block+inline?

Reply:
That... I don't know. When playing with containers before, I've tried both, and I think this is usually the one that's worked out well for me. Must be something I don't quite get, but it works here, so 🤷🏼 

(side note: if you read up on it and grok it proper, please let me know 😄 )